todolist/settings.py

```diff
@@ -62,12 +62,12 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
     'default': {
-        'ENGINE': 'mysql.connector.django',
-        'NAME': 'app_db',
-        'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': '172.17.0.2',  # You can use a different host if your MySQL server is on a remote machine.
-        'PORT': '',  # Leave this empty to use the default MySQL port (3306).
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'todolist_db',
+        'USER': 'user',
+        'PASSWORD': 'password',
+        'HOST': 'db',  # Назва сервісу MySQL у docker-compose.yml
+        'PORT': '3306',
```

Comment:
Database credentials are hardcoded, which is a security risk. Use environment variables to manage these credentials securely.